github/event_types.go

```diff
@@ -192,6 +200,10 @@ type DeploymentEvent struct {
 	// The following fields are only populated by Webhook events.
 	Sender       *User         `json:"sender,omitempty"`
 	Installation *Installation `json:"installation,omitempty"`
+
+	// The following field is only present when the webhook is triggered on
+	// a repository belonging to an organization.
+	Organization *Organization `json:"organization,omitempty"`
```

Comment:
Let's please name them all `Org`.

Reply:
I left the existing ones, as I wasn't sure if you wanted a breaking change here or not.  Happy to go change them all if you don't mind the breakage.

Reply:
Oh, sorry! That will teach me to do code reviews on my Android phone. 😂

Hmmm... consistency is nice as long as we document our changes.

Sure, let's be consistent and I'll mark this as a breaking API change. 😁

Reply:
Hmmm... I'm no longer on my phone, and I'm not understanding this statement:

> I left the existing ones, as I wasn't sure if you wanted a breaking change here or not. Happy to go change them all if you don't mind the breakage.

I dont see any existing `organization` fields in any of the structs you modified, so I don't think this is a breaking API change afterall, correct?